apps/builder/app/builder/features/style-panel/style-layer-utils.ts

```diff
@@ -181,3 +188,113 @@ export const swapLayers = (
   );
   batch.publish();
 };
+
+export const extractNameAndValueFromLayer = (
+  property: StyleProperty,
```

Comment:
why not using a list of specific properties?

Reply:
because right now you kinda claim to support them all from the interface and then throw an error if one passes you the wrong one